### PR TITLE
Created script for Defect 2432

### PR DESCRIPTION
**Pull request by TMyza**
ATF Test Scripts to check [#2432](https://github.com/smartdevicelink/sdl_core/issues/2432)

This PR is **ready** for review.

### Summary
A set of test scripts to verify that issue is fixed

### ATF version
[develop](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/5_0/2432_SDL_is_crashed_when_HMI_responds_DialNumber_value_is_not_string.lua [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+local function dialNumber(pParam)
+    local cid = common.getMobileSession():SendRPC("DialNumber", pParam)
+    common.getHMIConnection():ExpectRequest("BasicCommunication.DialNumber", pParam)
+    :Do(function(_, data)
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { info = 1111 })
+    end)
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+    :ValidIf(function(_, data)
+        if data.payload.info then
+            return false, "Unexpected parameter in SDL response"
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/2432_SDL_is_crashed_when_HMI_responds_DialNumber_value_is_not_string.lua]: Please define that unexpected parameter in response, and in what response:
```"Unexpected parameter 'info' in DialNumber response"```

**Comment by GetmanetsIrina** [comment]
PR is rebases on actual develop(a7db5cc) branch

**Comment by GetmanetsIrina** [comment]
Script is applicable for all policy flows and transport types.
Script has been moved to 6_1 folder.
Test set for script is added.

PTU with corresponding policies has been added.

**Comment by GetmanetsIrina** [comment]
PR is still relevant.

**Comment by GetmanetsIrina** [comment]
@jordynmackool, PR is ready for Livio review.
Please notice PR relates to the issue that was closed within the 6.1 Release.

**Review by iCollin — APPROVED** [review]

**Pull request merged**